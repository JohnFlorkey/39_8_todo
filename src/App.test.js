import { render, screen } from '@testing-library/react';
import App from './App';

// smoke test
test('renders learn react link', () => {
  render(<App />);
});

// snapshot test
it("should match snapshot", () => {
  const {asFragment} = render(<App />);
  expect(asFragment()).toMatchSnapshot();
})
