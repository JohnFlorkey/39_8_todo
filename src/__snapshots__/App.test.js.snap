// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="TodoList"
  >
    <form
      class="NewTodoForm"
    >
      <label
        class="NewTodoForm-label"
        for="task"
      >
        Task
      </label>
      <input
        class="NewTodoForm-input"
        id="task"
        name="task"
        type="text"
        value=""
      />
      <button>
        Add
      </button>
    </form>
    <div
      class="TodoList-todos"
    />
  </div>
</DocumentFragment>
`;
